Description: fix FTBFS caused after the DH level has bumped.
Author: Francisco Vilmar Cardoso Ruviaro <francisco.ruviaro@riseup.net>
Last-Update: 2020-06-20

--- a/configure.ac
+++ b/configure.ac
@@ -3,9 +3,9 @@ AC_INIT([canna], [3.7p3], [canna-dev@lis
 AC_CONFIG_SRCDIR([WHATIS.jp])
 AC_CONFIG_HEADER([accanna.h])
 
-if test "x$IN_MAKE" != xyes; then
-  AC_MSG_ERROR([You cannot invloke configure directly.])
-fi
+#if test "x$IN_MAKE" != xyes; then
+#  AC_MSG_ERROR([You cannot invloke configure directly.])
+#fi
 
 # Checks for programs.
 AC_PROG_CC
